src/coinbase/address/wallet_address.ts

```diff
@@ -378,6 +406,32 @@ export class WalletAddress extends Address {
     return SmartContract.fromModel(resp?.data);
   }
 
+  /**
+   * Creates an ERC721 token contract.
+   *
+   * @param options - The options for creating the ERC721 token.
+   * @param options.name - The name of the ERC721 token.
+   * @param options.symbol - The symbol of the ERC721 token.
+   * @param options.baseURI - The base URI of the ERC721 token.
+   * @returns A Promise that resolves to the deployed SmartContract object.
+   * @throws {Error} If the private key is not loaded when not using server signer.
```

Comment:
Lets document this may throw an `APIError`

Reply:
good catch fixed